frontend/context/BalanceProvider.tsx

```diff
@@ -285,22 +293,23 @@ const getCrossChainWalletBalances = async (
 
 const getCrossChainStakedBalances = async (
   services: MiddlewareServiceResponse[],
+  masterSafeAddress: Maybe<Address>,
 ): Promise<CrossChainStakedBalances> => {
   const result: CrossChainStakedBalances = [];
 
   const registryInfoPromises = services.map(async (service) => {
     const serviceConfigId = service.service_config_id;
     const middlewareChain: MiddlewareChain = service.home_chain;
     const chainConfig = service.chain_configs[middlewareChain];
-    const { multisig, token } = chainConfig.chain_data;
+    const { token: serviceNftTokenId } = chainConfig.chain_data;
 
-    if (!multisig || !token) {
+    if (!masterSafeAddress || !serviceNftTokenId) {
       return null;
     }
 
     const registryInfo = await StakedAgentService.getServiceRegistryInfo(
-      multisig,
-      token,
+      masterSafeAddress,
```

Comment:
`getOperatorBalance` should use masterSafe address, not service multisig, that's why the bond part of staked balance was 0